pages/windows/msg.md

```diff
@@ -0,0 +1,23 @@
+# msg
+
+> Send a message to a specific user or session.
+
+- Send a message to a specified user or session:
+
+`msg {{username|session_name|session_id}} {{message}}`
+
+- Send a message from stdin:
+
+`echo "{{message}}" | msg`
+
+- Send a message to a specific server:
+
+`msg /server:{{server_name}} {{username|session_name|session_id}}`
+
+- Send a message to all users:
```

Comment:
Maybe clarify that this sends to all users of the current machine ? Since this example is right after an example to send to a specific server.